Userland/Libraries/LibX86/Instruction.cpp

```diff
@@ -1343,46 +1366,33 @@ static String sib_to_string(u8 rm, u8 sib)
 String MemoryOrRegisterReference::to_string_a64() const
 {
     if (is_register())
-        return register_name(static_cast<RegisterIndex32>(m_register_index));
+        return register_name(static_cast<RegisterIndex64>(m_register_index));
 
     bool has_displacement = false;
     switch (mod()) {
+    case 0b00:
+        has_displacement = m_rm == 5;
+        break;
```

Comment:
Didn't you just add this function in one of the previous commits,
Maybe fix it up there.